src/main/java/com/commercetools/sync/inventories/InventoryService.java

```diff
@@ -0,0 +1,65 @@
+package com.commercetools.sync.inventories;
+
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.inventory.InventoryEntry;
+import io.sphere.sdk.inventory.InventoryEntryDraft;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Provides CRUD operations upon {@link InventoryEntry}.
+ */
+interface InventoryService {
```

Comment:
@heshamMassoud see conversation under this review: https://github.com/commercetools/commercetools-sync-java/pull/19#pullrequestreview-34777373